src/ui/components/Rating/index.js

```diff
@@ -79,6 +79,7 @@ export class RatingBase extends React.Component {
         <button
           onClick={this.onSelectRating}
           value={thisRating}
+          title="Rate this add on %(thisRating) out of 5."
```

Comment:
That's not what you'll want to do; that syntax is only for `gettext` strings. In fact, what you'll want to do here is add a `title` property to each rating above.

Reply:
(At line 58 here, you'll want to add a title attribute with `Rate this add on ${thisRating} out of 5.`... as long as the addon is not `readOnly`.)